fastlane/lib/fastlane/tools.rb

```diff
@@ -0,0 +1,20 @@
+module Fastlane
+  TOOLS = [
+    :fastlane,
+    :pilot,
+    :spaceship,
+    :produce,
+    :cert,
+    :deliver,
+    :frameit,
+    :pem,
+    :snapshot,
+    :screengrab,
+    :supply,
+    :cert,
+    :sigh,
+    :match,
+    :scan,
+    :gym
+  ]
```

Comment:
You can use `%i{fastlane pilot ...}` (since Ruby 2.0) if you like the way that reads better 👍